container/openejb-core/src/main/java/org/apache/openejb/config/ActivationConfigPropertyOverride.java

```diff
@@ -130,6 +142,34 @@ public AppModule deploy(final AppModule appModule) throws OpenEJBException {
         return appModule;
     }
 
+    private MdbContainer getMdbContainer(final String containerId) {
+
+        final ContainerSystem containerSystem = SystemInstance.get().getComponent(ContainerSystem.class);
+
+        if (containerSystem == null) {
```

Comment:
@rmannibucau 
If I remove this.
It'll break the ActivationConfigPropertyOverrideTest with NPE